src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -96,19 +91,32 @@ export class PmdCatalogWrapper extends PmdSupport {
 		return parameters;
 	}
 
-	private addPmdJarPaths(rulePathEntries: Map<string, Set<string>>): void {
+	/**
+	 * Return a map where the key is the language and the value is a set of class/jar paths.  Start with the given
+	 * default values, if provided.
+	 */
+	private addPmdJarPaths(defaults?: Map<string, Set<string>>): Map<string, Set<string>> {
```

Comment:
It's weird to me that `defaults` is the parameter name, but it actually represents the custom rules. Or maybe I'm just misunderstanding.

Reply:
Yeah, that was a weird method.  I refactored it to be much cleaner: just getRulePathEntries() now.